templates/log4j2.xml.erb

```diff
@@ -24,9 +24,16 @@
             </Policies>
         </RollingFile>
 <%- elsif @log_local_policy == 'size' -%>
-        <RollingFile name="defaultFile"
+          <%- if @log_file_perm == 'public' -%>     
+           <RollingFile name="defaultFile"
+            filename="<%= @log_dir-%>/<%= @app_name -%>.log"
+            filePattern="<%= @log_dir-%>/<%= @app_name -%>.log.%i"
+            filePermissions="rw-r--r--">
```

Comment:
to prevent code duplication, if statement should only wrap the single parameter

Reply:
I thinks this duplicates even more. ```if``` statements can be nested.  All that is required is wrap the ```filePermissions``` parameter in an ```if`` block. templates/keystone-v2.cfg.xml.erb has many examples of this